cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -73,16 +101,16 @@ def import_handler(request, course_key_string):
             # Do everything in a try-except block to make sure everything is properly cleaned up.
             try:
                 data_root = path(settings.GITHUB_REPO_ROOT)
-                course_subdir = "{0}-{1}-{2}".format(course_key.org, course_key.course, course_key.run)
-                course_dir = data_root / course_subdir
+                subdir = base64.urlsafe_b64encode(repr(courselike_key))
```

Comment:
@doctoryes I don't see why this would break any existing courses-- this is a temporary directory used for the import, and it should be destroyed after everything is loaded in.